### PR TITLE
fix #25

### LINKED ISSUE #25: Ignore main packages
Breaking changes in a main package aren't breaking changes (they can't be imported).

### PR DESCRIPTION
**Pull request by hydroflame**

### CODE REVIEW

File: apicompat.go [hunk 1]
```diff
@@ -217,15 +225,13 @@ func (c Checker) getDirsRecursive(base, rev, rel, prefix string) (dirs []string)
...
 	// Use go/build to get the list of files relevant for a specific OS and ARCH
-
 	ctx := build.Default
 	ctx.ReadDir = func(dir string) ([]os.FileInfo, error) {
 		return c.vcs.ReadDir(rev, dir)
 	}
 	ctx.OpenFile = func(path string) (io.ReadCloser, error) {
 		return c.vcs.OpenFile(rev, path)
 	}
-	ctx.GOPATH = os.Getenv("GOPATH")
```
Comment (bradleyfalzon) [inline apicompat.go]: This'll be why the tests are failing.

File: apicompat.go [hunk 2]
```diff
@@ -24,6 +24,11 @@ import (
 // go/build understands this as "."
 const cwd = "."
 
+var (
+	// ErrSkipPackage is returned by the parser when a package should be skipped.
+	ErrSkipPackage = errors.New("Skipping package")
```
Comment (bradleyfalzon) [inline apicompat.go]: Should we leave it unexported, the function that uses it isn't exported, although I haven't had a good look just yet, I don't think other libraries would need access to it

File: apicompat.go (hunk 1 above)
Comment (hydroflame) [inline apicompat.go]: I believe you, but that's pretty weird considering 
https://github.com/golang/go/blob/master/src/go/build/build.go#L265

**Comment by bradleyfalzon** [comment]
Brilliant, thanks. I've disabled the coveralls comments and status api, not exceedingly useful.

**Pull request merged**